daemon/cmd/daemon_main.go

```diff
@@ -1138,6 +1138,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.MarkHidden(option.EnableK8sNetworkPolicy)
 	option.BindEnv(vp, option.EnableK8sNetworkPolicy)
 
+	flags.Bool(option.PolicyCIDRSelectsNodes, defaults.PolicyCIDRSelectsNodes, "enable selecting in-cluster nodes via NetworkPolicy CIDR selectors")
```

Comment:
Seems reasonable. Right now, `--policy-cidr-match-mode=remote-node` would not work, since I'm not about to go disabling the CIDR path. But I could imagine a future wherein that does work.

I'll add that flag and patch up the config logic.

Reply:
This is done.